src/ControlSystem/Measurements/BNSCenterOfMass.hpp

```diff
@@ -16,6 +16,7 @@
 #include "Domain/Structure/ObjectLabel.hpp"
 #include "NumericalAlgorithms/LinearOperators/DefiniteIntegral.hpp"
 #include "Parallel/Reduction.hpp"
+#include "ParallelAlgorithms/Events/Tags.hpp"
```

Comment:
Do we want to instead declare all Events::Tags::ObserverXXX tags in the header?

Reply:
Up to you. We already have to depend on the `Events` library for `Event.hpp` in the control system so it doesn't really matter.

Reply:
Ok, great; I'm happy with it as is.